controllers/catsController.js

```diff
@@ -44,5 +44,18 @@ router.put("/api/cats/:id", function(req, res) {
   });
 });
 
+router.delete("/api/cats/:id", function(req, res) {
+  var condition = "id = " + req.params.id;
+
+  cat.delete(condition, function(result) {
+    if (result.affectedRows == 0) {
+      // If no rows were changed, then the ID must not exist, so 404
+      return res.status(404).end();
+    } else {
+      res.status(200).end();
+    }
+  });
+});
+
```

Comment:
The `catsController.js` file to have a `/api/cats/:id` delete route, to call the delete key of the cat model, and to pass in arguments as necessary